### PR TITLE
Feature/stm32g crs

### PR DESCRIPTION
**Pull request by kevswims**
Adds the CRS peripheral to the STM32 G0 and G4. Previously this was only used on the L0.

### CODE REVIEW

File: stm32-data-gen/src/chips.rs [hunk 1]
```diff
@@ -284,7 +284,9 @@ impl PeriMatcher {
             ("STM32H50.*:EXTI:.*", ("exti", "h50", "EXTI")),
             ("STM32H5.*:EXTI:.*", ("exti", "h5", "EXTI")),
             (".*:EXTI:.*", ("exti", "v1", "EXTI")),
-            ("STM32L0.*:CRS:.*", ("crs", "l0", "CRS")),
+            ("STM32L0.*:CRS:.*", ("crs", "v1", "CRS")),
```
Comment (kevswims) [inline stm32-data-gen/src/chips.rs:287]: Note: I couldn't find an actual silicon version for these. Is v1 the correct thing to label this?

File: stm32-data-gen/src/chips.rs [hunk 2]
```diff
@@ -284,7 +284,9 @@ impl PeriMatcher {
...
+            ("STM32G0.*:CRS:.*", ("crs", "v1", "CRS")),
```
Comment (kevswims) [inline stm32-data-gen/src/chips.rs:288]: I'm not quite sure if all G0 chips have the CRS peripheral. Is there a better way to constrain this down or does it matter?

File: stm32-data-gen/src/chips.rs (hunk 1 above)
Comment (Dirbaio) [reply stm32-data-gen/src/chips.rs:287]: yeah, that's fine. Most v1/v2/v3... is invented by us in roughly chip release chronological order.

**Comment by Dirbaio** [comment]
Thank you!

**Pull request merged**